docs/reference/ilm/apis/put-lifecycle.asciidoc

```diff
@@ -33,7 +33,8 @@ include::{docdir}/rest-api/timeoutparms.asciidoc[]
 
 ==== Authorization
 
-include::ilm-cluster-mgt-privilege.asciidoc[]
+You must have the `manage_ilm` cluster privilege to use this API.
+For more information, see {stack-ov}/security-privileges.html[Security Privileges].
```

Comment:
I think we should also explain here that the user putting the policy must have permissions to run the APIs for the actions on any indexes that will be used by the policy

Reply:
agreed